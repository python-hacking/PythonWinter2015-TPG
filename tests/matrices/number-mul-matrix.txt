f = 2.2
f * [[ 1 3] [0.1 9]]
