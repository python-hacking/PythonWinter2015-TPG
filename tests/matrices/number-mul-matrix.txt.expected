[
 [2.2 6.6]
 [0.22 19.8]
]
